src/UseCases/GetInTouch/GetInTouchUseCase.php

```diff
@@ -24,17 +25,17 @@ public function __construct( GetInTouchValidator $validator, Messenger $messenge
 		$this->messenger = $messenger;
 	}
 
-	public function processContact( GetInTouchRequest $request ): string {
+	public function processContact( GetInTouchRequest $request ): ValidationResponse {
 		$this->request = $request;
 
 		if ( !$this->validator->validate( $request ) ) {
-			return 'validation failed';
+			return ValidationResponse::newFailureResponse( $this->validator->getConstraintViolations() );
 		}
 
 		try {
 			$this->forwardContactRequest();
 			$this->confirmToUser();
-			return 'request successful';
+			return ValidationResponse::newSuccessResponse();
 		} catch ( \RuntimeException $e ) {
 			return 'mail transmission failed';
```

Comment:
This will create a TypeError because a string is returned. Either don't catch the error (it will be handled by the default error handler) or create a new response.
Since there is nothing much the user can do in this case (and we don't need to inform him what failed specifically), I'd recommend not handling the exception and displaying the general error response.